### ISSUE TITLE
Onboarding Video

### ISSUE THREAD

**Issue by jessicazartler** [CommonsBuild/coordination#2]
## 🎉   Subtasks

- [x] Define the Production team
- [ ] Write the Storyboard / Concept
- [ ] Look for inspiration from other videos around the web
- [ ] Write a script
- [ ] Decide on the production approach based on the resources available
- [ ] Music selection
- [ ] Video editing
- [ ] Reviews
- [ ] Distribute and promote

## 🤼‍♂️   Reviewer

@

**Comment by jessicazartler** [comment]
Comments/feedback from comms call on what any onboarding video should consider:

:Hard to follow, so much info, put our newcomer hat on and think from that 
Perspective, really important as this is a key communication
:International - legal conditions different - maybe different videos/localize
:Who/what we are communicating with - WHY we are here
	:Specific/focused videos - how to make a proposal or how to invest in the hatch
	:Onboarding into WG video - and a few actions CTAs
	:Onboarding process as narrative arc
	:Opportunity to work for crypto / earn without needing an exchange - alternative 
working your way in instead of buying your way in
	:Inspiring, revealing a way they can get involved what you can do and how to

**Comment by jessicazartler** [comment]
This is an epic - need to identify issues underneath and assign them @tamarandom if you can help me -- looking like we'll prob need a session to organize the Comms board.. Will DM Thank you!

**Comment by tamarandom** [comment]
🙏🏼  Today or tomorrow. 🙏🏼

**Comment by markoprljic** [comment]
1. Jess will start with Interviews
2. Draft Storyboard doc
3. Share with the group for more ideas

**Comment by mount-manu** [comment]
@chuygarcia92 I can help with this. I've been (secretly) onboarding people since ETHCC.

**Comment by natenatesoo** [comment]
Update: Still work in progress.

**Comment by tamarandom** [comment]
@mount-manu @chuygarcia92 Is this video different than the Community video that was just made?

**Comment by tamarandom** [comment]
Update: closing this. There is new initiative to do orientation video. 

Tagging @chuygarcia92 and @danelsuga to look at the original idea.
